app/views/idv/shared/_document_capture.html.erb

```diff
@@ -18,10 +18,10 @@
   mock_client: (DocAuthRouter.doc_auth_vendor == 'mock').presence,
   document_capture_session_uuid: flow_session[:document_capture_session_uuid],
   endpoint: FeatureManagement.document_capture_async_uploads_enabled? ?
-    idv_doc_auth_step_path(step: :verify_document) :
+    send(@step_url, step: :verify_document) :
```

Comment:
reason this is needed:

In the non-hybrid flow, we link to `/idv/doc_auth` etc endpoints. However, in the hybrid flow, the mobile path hits `/idv/capture_doc/` versions of the corresponding endpoints, so this needed to be updated to be dynamic based on the flow